portfolio/src/main/webapp/script.js

```diff
@@ -13,14 +13,15 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
+      ['I was born on April 24th, 2000.', 'I learned Spanish before I learned English.', 'I have not cut the length of my hair since I graduated high school two years ago.', 
+      'I was named after Sean Connery, but I do not believe I have ever seen any of his movies.'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  // Pick a random fact.
+  const greeting = facts[Math.floor(Math.random() * facts.length)];
 
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
```

Comment:
Use `querySelector` now. `getElementById` works, but `querySelector('#greeting-container')` is preferred.